Gemfile

```diff
@@ -28,3 +28,5 @@ gem 'activesupport'
 gem 'pygments.rb'
 # gem 'nokogiri'
 # gem 'iconv'
+gem 'tzinfo-data'
```

Comment:
Why these new gems are added here?

Reply:
When I run the jekyll command, it showed error for these gems, Thus I added them.

Reply:
Can you paste the error here?

Reply:
![image](https://github.com/shireeshj/blog/assets/37475140/dbaea90b-7336-46bb-bd55-6330a975c71b)
C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/tzinfo-1.2.11/lib/tzinfo/data_source.rb:182:in `rescue in create_default_data_source': No source of timezone data could be found. (TZInfo::DataSourceNotFound)
Please refer to https://tzinfo.github.io/datasourcenotfound for help resolving this error.

Reply:
sure will do.